package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "bootstrap": "^4.5.3",
```

Comment:
👋 Liz, remember that we should not use any CSS frameworks moving forward until we get to the last 2 weeks of work 😅.

**It's ok to leave it like this**, but don't spend much time on this as the style might change completely in the end.